src/pages/CompanyTrainingOverview/CTOPage.jsx

```diff
@@ -0,0 +1,97 @@
+// Import necessary components from Material-UI library
+import Button from "@mui/material/Button";
+import { List, ListItem, ListItemText } from "@mui/material";
+// Import custom PageLayout component
```

Comment:
You dont have to comment ever line of code

Reply:
Comments removed